code/modules/urist/mob/uristanimals.dm

```diff
@@ -164,4 +164,172 @@ Please keep it tidy, by which I mean put comments describing the item before the
 
 	if(!reagents)
 		create_reagents(chem_type_amt * 10)
-	generate_venom(chem_type_amt)
\ No newline at end of file
+	generate_venom(chem_type_amt)
+
+
+	//geese from Bay
+/mob/living/simple_animal/hostile/retaliate/goose
+	name = "goose"
+	desc = "A large waterfowl, known for its beauty and quick temper when provoked."
+	icon = 'icons/uristmob/goose.dmi'
+	icon_state = "goose"
+	icon_living = "goose"
+	icon_dead = "goose_dead"
+	speak = list("honk")
+	speak_emote = list("honks")
+	emote_see = list("flaps its wings")
+	speak_chance = 1
+	turns_per_move = 5
+	response_help =  "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "strikes"
+	health = 45
+	maxHealth = 45
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	pass_flags = PASS_FLAG_TABLE
+	faction = "geese"
+	attacktext = "strikes"
+	pry_time = 8 SECONDS
+	break_stuff_probability = 5
+
+
+	var/enrage_potency = 3
+	var/enrage_potency_loose = 4
+	var/loose_threshold = 15
+	var/max_damage = 22
+
+
+
+/mob/living/simple_animal/hostile/retaliate/goose/death(gibbed, deathmessage, show_dead_message)
+	. = ..()
+	update_icon()
+
+/mob/living/simple_animal/hostile/retaliate/goose/proc/enrage(potency)
+//	var/obj/item/W = get_natural_weapon()
+		health = (initial(health) * 1.5)
```

Comment:
This will have a weird effect of auto-healing the goose on Enrage.

Reply:
removed that line